tasks/javascript.js

```diff
@@ -105,18 +113,26 @@ gulp.task('javascript:lint', function () {
 
 // karma unit tests in local library only
 gulp.task('javascript:unit', function () {
-  return streams.jsSpec()
-    .pipe(browserify
-      .compile(80, transforms.concat(browserify.jasmineTransform('@')))
-      .all('index.js'))
+  return combined.create()
+    .append(
+      streams
+        .testDependencies({
+          dev: true,
+          read: false
+        })
+    )
+    .append(
+      streams
+        .jsSpec()
+        .pipe(browserify
+          .compile(80, transforms.concat(browserify.jasmineTransform('@')))
+          .all('index.js'))
+        .pipe(gulpFilter('*.js'))
```

Comment:
place the filter after the `dest()` so that we get the source map in the directory, for consistency